src/include/access/heapam.h

```diff
@@ -121,6 +121,7 @@ extern TableScanDesc heap_beginscan(Relation relation, Snapshot snapshot,
 extern void heap_setscanlimits(TableScanDesc scan, BlockNumber startBlk,
 							   BlockNumber numBlks);
 extern void heapgetpage(TableScanDesc scan, BlockNumber page);
+extern void heap_only_get_page(HeapScanDesc scan, BlockNumber page);
```

Comment:
could you rename this to `heap_getpageonly` to match with the naming convention in this file? (put "only" at the end to avoid confusion with the "heap only tuple", which is somewhat unrelated).